Source/LibationAvalonia/Dialogs/Login/LoginChoiceEagerDialog.axaml

```diff
@@ -60,7 +60,7 @@
 
 			<controls:LinkLabel
 				Tapped="ExternalLoginLink_Tapped"
-				Text="Or click here to log in with your browser." />
+				Text="Trouble logging in? Click here to log in with your browser." />
```

Comment:
I've been calling it the 'or click here' link for so long. I wonder how many times I'll call this the wrong thing